test/core/ticktree/TickTreeCleanOffer.t.sol

```diff
@@ -27,13 +27,13 @@ import "mgv_lib/Debug.sol";
 //     3. the offer to be cleaned is last of two offers
 //     4. the offer to be cleaned is middle of three offers
 // - higher bin list
-//   - bin has higher position in same leaf or level2-3 as ToI
+//   - bin has higher position in same leaf or level3-3 as ToI
 //     - if feasible, given cleaning tick
 //   - list:
 //     1. is empty
 //     2. is non-empty
 // - lower bin list
-//   - bin has lower position in same leaf or level2-3 as ToI
+//   - bin has lower position in same leaf or level3-3 as ToI
```

Comment:
```suggestion
//   - bin has lower position in same leaf or level1-3 as BoI
```

Reply:
fixed in b629e86b9d32fcd905bf8643ef65408841751c08